src/main/java/uk/co/evoco/webdriver/configuration/WebDriverConfig.java

```diff
@@ -184,20 +172,39 @@ public void setRunType(String runType) {
     }
 
     /**
-     *
+     * @return exceptionsWaitTimeOut if user specified the exception time out in the config file otherwise
```

Comment:
Follow javadocs convention; should be a blank line here

Reply:
added the blank line